### ISSUE TITLE
Warn on bad COPY last argument

### ISSUE THREAD

**Issue by lorenzo** [hadolint/hadolint#154]
- [ ] This is a bug report
- [x] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

hadolint should warn that last argument to `COPY` when using more than 2 arguments should end in a `/`

### Actual behavior

No warning, but `docker build` fails

```
Step 1/1 : COPY ./stack.yaml ./package.yaml .
When using COPY with more than one source file, the destination must be a directory and end with a /
```

### Steps to reproduce the behavior
```
COPY a_file b_file .
```
